### DISCUSSION TITLE
Help regarding project on Decentralized authentication and authorization in cloud environment

### DISCUSSION THREAD

**Discussion by kirangeet** [Cyfrin/foundry-full-course-cu#1684] [General]
hi friends 
Trying to create a SSO kind of project  can I get some ideas or links of already created projects

**Comment by EngrPips** [comment]
Hello @kirangeet, What is SSO?

**Reply by kirangeet** [reply]
It means single sign on 
I want that if user register on one site he can login to another site without registering 
Like we can  login to discord using Gmail

**Reply by EngrPips** [reply]
Oh, Interesting. I have yet to build out such a thing, but I trust some competent people would have some advice for you. You can browse Google and continue personal research while you wait for a response here.
